tools/public_api_guard/youtube-player/youtube-player.d.ts

```diff
@@ -1,15 +1,15 @@
 export declare class YouTubePlayer implements AfterViewInit, OnDestroy, OnInit {
-    apiChange: EventEmitter<YT.PlayerEvent>;
+    apiChange: Observable<YT.PlayerEvent>;
```

Comment:
This is technically a public API change, but `Observable` and `EventEmitter` have the same signature, aside from `emit` which people shouldn't be accessing on our outputs anyway.